testsuite/features/secondary/srv_cobbler_distro.feature

```diff
@@ -18,11 +18,13 @@ Feature: Cobbler and distribution autoinstallation
 
   Scenario: Ask cobbler to create a distribution via API
     Given cobblerd is running
+    # TODO: Move to SUMA API
```

Comment:
yup. I don't think we really need a comment here, since we have a card for it. But as you prefer.

Reply:
I would like to leave the comment. I checked the Cobbler distro, profile and sync features and those 2 are the only steps where we use the Cobbler API directly in the step. Of course, there are other places where we use it, too. But I think this will help us in a few months.